src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/TodoTemplate.Api.csproj

```diff
@@ -6,6 +6,7 @@
         <MvcRazorExcludeRefAssembliesFromPublish>false</MvcRazorExcludeRefAssembliesFromPublish>
         <PreserveCompilationReferences>true</PreserveCompilationReferences>
         <PreserveCompilationContext>true</PreserveCompilationContext>
+        <UserSecretsId>df8517d0-6b4c-46e5-a8b6-fe97ed6b0e96</UserSecretsId>
```

Comment:
this line seems redundant, shouldn't it be removed? or did you intend to use the secret manager somehow?

Reply:
Remove it please @milad-karimifard